examples/file-feed/complete/src/browser-app/index.css

```diff
@@ -0,0 +1,24 @@
+@import url('https://fonts.googleapis.com/css?family=Anonymous+Pro|Source+Code+Pro');
```

Comment:
Sure. My thinking is when the connection to `googleapis.com` is bad or completely missing and the UI would be designed for those fonts to exists. When they fail to load, the page would default to using some other font and the layout wouldn't work the same way anymore, could lead to unexpected shuffling around the elements.